docs/src/misc.rst

```diff
@@ -461,6 +461,17 @@ API
 
     Gets memory information (in bytes).
 
+.. c:function:: uint64_t uv_get_constrained_memory(void)
+
+    Gets the amount of memory available to the process (in bytes) based on
+    limits imposed by the OS. If there is no such constraint, or the constraint
+    is unknown, `0` is returned. Note that it is not unusual for this value to
+    be less than or greater than :c:func:`uv_get_total_memory`.
+
```

Comment:
Can you add a `.. versionadded::`. Just search this file for an example. The version will be 1.29.0.